source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -75,12 +81,26 @@
      - ⊛
      - ⊛
      - ⊛
+     - ⊛
+     - ✓
+     - ✓
+     - ✓
      - ✓
+
+   * - 4.0
+     - ⊛
+     - ⊛
+     - ⊛
+     - ⊛
+     - ⊛
+     - ⊛
+     - ⊛
      - ✓
      - ✓
      - ✓
 
-   * - 3.11 to 4.0
+   * - 3.11 to 3.12
+     - X [#v3-note]_
```

Comment:
```suggestion
     - ✗ [#v3-note]_
```
If ✓ is U+2713 the "X" should be stylized similarly shouldn't it? Maybe U+2717 (✗)